src/components/Comments/WriteComment.js

```diff
@@ -170,30 +170,30 @@ export const styles = StyleSheet.create({
     borderTopColor: COLORS.gray1,
     position: 'absolute'
   },
-  replyToContainer: (inReplyCommentView) => ({
-    marginLeft: inReplyCommentView ? 90 : 60,
+  replyToContainer: {
+    marginLeft: 60,
     fontFamily: fonts.inter[600],
     marginBottom: 11,
     marginTop: 7,
     lineHeight: 15,
     fontSize: 12,
     color: COLORS.blackgrey
-  }),
+  },
   replyToTitle: {
     fontFamily: fonts.inter[600],
     lineHeight: 15,
     fontSize: 12,
     color: COLORS.black
   },
-  container: (inReplyCommentView) => ({
+  container: {
     flex: 1,
     backgroundColor: COLORS.white,
     width: '100%',
     paddingRight: 20,
-    paddingLeft: inReplyCommentView ? 43 : 20,
+    paddingLeft: 20,
     flexDirection: 'row',
     zIndex: 100
-  }),
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here remove the dynamic styling based on `inReplyCommentView`. This could potentially lead to UI inconsistencies if `inReplyCommentView` was used to adjust the layout under certain conditions. Please verify that these changes do not negatively impact the user interface.

```diff
- replyToContainer: (inReplyCommentView) => ({
-   marginLeft: inReplyCommentView ? 90 : 60,
-   ...
- }),
- container: (inReplyCommentView) => ({
-   paddingLeft: inReplyCommentView ? 43 : 20,
-   ...
- }),

+ replyToContainer: {
+   marginLeft: 60,
+   ...
+ },
+ container: {
+   paddingLeft: 20,
+   ...
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->